src/libraries/Microsoft.PowerFx.Core/Parser/ParserOptions.cs

```diff
@@ -20,18 +20,39 @@ public class ParserOptions
         /// </summary>
         public bool AllowsSideEffects { get; set; }
 
+        /// <summary>
+        /// The culture that an expression will parse with. 
+        /// This primarily determines numeric decimal separator character
+        /// as well as chaining operator. 
+        /// </summary>
         public CultureInfo Culture { get; set; }
 
+        /// <summary>
+        /// If greater than 0, enforces a maximum length on a single expression. 
+        /// It is an immediate parser error if the expression is too long. 
+        /// </summary>
+        public int MaxExpressionLength { get; set; }
```

Comment:
>set [](http://example.com/codeflow?start=46&length=3)

test the incoming value here

Reply:
I have to check it against the text field. So I can't do in an individual setter - must be done at apply operation.